api/views.py

```diff
@@ -160,3 +164,31 @@ def test(request):
         "status": "success",
         "payload": "Setup successful!"
     }, status=200)
+
+
+@api_view(["GET"])
+def rpms_images_fetcher_view(request):
+    release = request.query_params.get("release", None)
+
+    if release is None:
+        return Response(data={"status": "error", "message": "Missing \"release\" params in the url."})
+
+    # Try to fetch data from cache
+    cache_key = f"rpms_images_data_{release}"
```

Comment:
let's not use cache here, image list may changed while image adding/removing in upstream, cache content may incorrect